doc/MANUAL.adoc

```diff
@@ -463,6 +463,16 @@ might be incorrect.
 
     If set to a file path, ccache will write information on what it is doing to
     the specified file. This is useful for tracking down problems.
++
+If set to "syslog", then ccache will log using syslog() instead of a file.
+Then you can use syslogd configuration to filter into a file, example:
```

Comment:
I think that it would be a good idea to mention /etc/rsyslog.conf and /etc/rsyslog.d here. Perhaps like this?

```
If set to *syslog', ccache will log using `syslog()` instead of to a file. If
you use rsyslogd, you can add something like this to `/etc/rsyslog.conf` or a
file in `/etc/rsyslog.d`:
```

Reply:
Not sure it was rsyslogd specific, but I suppose that the file was indeed intended for /etc/rsyslog.d